chassis/configuration/configuration/reporter/session.comparison.conf.xml

```diff
@@ -77,13 +61,6 @@
 
     <!-- Monitoring -->
     <!-- Deprecated. Not used starting from 1.2.5 -->
-    <bean id="monitoringFeatureComparator"
-          class="com.griddynamics.jagger.engine.e1.sessioncomparation.monitoring.MonitoringFeatureComparator">
-
-        <property name="monitoringParameterDecisionMaker" ref="stdDevMonitoringParameterDecisionMaker"/>
-        <property name="monitoringSummaryRetriever" ref="monitoringSummaryRetriever"/>
-        <property name="sessionFactory" ref="sessionFactory"/>
-    </bean>
     <bean id="stdDevMonitoringParameterDecisionMaker"
           class="com.griddynamics.jagger.engine.e1.sessioncomparation.monitoring.StdDevMonitoringParameterDecisionMaker">
 
```

Comment:
in this XML
following code is useless
```
 <alias name="defaultSessionComparator" alias="sessionComparator"/>
    <bean id="defaultSessionComparator"
          class="com.griddynamics.jagger.engine.e1.sessioncomparation.ConfigurableSessionComparator">
        <property name="comparatorChain">
            <ref bean="comparators_${chassis.decision.maker.with.limits.is.only.decision.maker}"/>
        </property>
        <property name="decisionMaker" ref="worstCaseDecisionMaker"/>
    </bean>
    <bean id="worstCaseDecisionMaker"
          class="com.griddynamics.jagger.engine.e1.sessioncomparation.WorstCaseDecisionMaker"/>

    <util:list id="comparators_true">
        <ref bean="limitsBasedFeatureComparator"/>
    </util:list>

    <util:list id="comparators_false">
        <ref bean="workloadFeatureComparator"/>
        <ref bean="limitsBasedFeatureComparator"/>
    </util:list>
```

\+ chassis.decision.maker.with.limits.is.only.decision.maker will be always true => can be deleted